src/style.css

```diff
@@ -1,3 +1,102 @@
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-size: 12px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  height: 100%;
+  margin-top: 0;
+}
+
+.nav {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
This class styles is redundant. You can reset default styles at the begining of the file by using tag selectors. I have  mentioned it in prev comment.